libs/@local/harpc/client/typescript/src/wire-protocol/stream/ResponseFromBytesStream.ts

```diff
@@ -84,6 +84,20 @@ export const make = <E, R>(
 
   return pipe(
     stream,
+    Stream.tap((chunk) => {
```

Comment:
i am in the process of authoring a PR in effect that'd allow us to do something similar to the [`enabled!` ](https://docs.rs/tracing/latest/tracing/macro.enabled.html)macro in tracing (gate this somewhat expensive computation - and others - to only when enabled)

Reply:
For the time being we keep this code or do you want to wait?

Reply:
I'd keep it, there's a bit of performance penalty, but it isn't too large.